### DISCUSSION TITLE
Welcome to TauonMusicBox Discussions!

### DISCUSSION THREAD

**Discussion by Taiko2k** [Taiko2k/Tauon#428] [General]
## 👋 Welcome!

This looks cool. An alternative to creating issues. 

Feel free to ask or post stuff!

**Comment by Solarunit** [comment]
Cool!
Just want to thank you again for the awesome music player! Great work, I love it! :heart: :musical_note: :headphones:

**Reply by Taiko2k** [reply]
Thanks Solarunit! Glad to hear you still like it after all this time. :heart:

**Reply by Solarunit** [reply]
Sure, its number 1 Linux app for me :wink:
